src/components/Hero/Hero.mdx

```diff
@@ -131,10 +131,19 @@ subHeaderText={otherSubHeaderText}
 
 ### Campaign
 
-The `"campaign"` hero type can be used with the `backgroundColor`,
-`backgroundImageSrc`, `foregroundColor`, `heading`, `imageProps`, and
-`subHeaderText` props. The minimum props to use are `backgroundImageSrc`,
-`heading`, and `imageProps`.
+The `"campaign"` hero type can be used with the `backdropBackgroundColor`,
+`backgroundColor`, `backgroundImageSrc`, `foregroundColor`, `heading`,
+`imageProps`, and `subHeaderText` props. The minimum props to use are
+`backgroundImageSrc`, `heading`, and `imageProps`.
+
+This variant has the special use case of having a background and foreground. The
+"backdrop" is what we call the background and that can be set through either the
+`backdropBackgroundColor` or `backgroundImageSrc` props. When both are set, the
+`backgroundImageSrc` will take precedence.
+
+To modify the foreground, the `background` and `imageProps` props should be used.
+View the last example in this section to see an example of the backdrop and
+foreground settings in action.
```

Comment:
This paragraph should also mention `foregroundColor`.  And maybe mention that we understand that the prop usage is a little confusing.